### PR TITLE
Fix windows integration tests

### PR DESCRIPTION
**Pull request by antontroshin**
# Description

etcd keys path differs between OS

## Issue reference

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by cicoyle — APPROVED** [review]

**Comment by cicoyle** [comment]
Changes look good. I think you just need to run ‘make lint’

File: tests/integration/suite/actors/reminders/scheduler/remove.go [hunk 1]
```diff
@@ -142,14 +140,14 @@ func (r *remove) Run(t *testing.T, ctx context.Context) {
...
-		keys, rerr := etcdClient.ListAllKeys(ctx, "dapr/jobs")
+		keys, rerr := etcdClient.ListAllKeys(ctx, etcdKeysPrefix)
 		require.NoError(c, rerr)
 		assert.Len(c, keys, 1)
 	}, time.Second*10, 10*time.Millisecond)
 
 	assert.EventuallyWithT(t, func(c *assert.CollectT) {
-		assert.Equal(c, int64(1), r.triggered.Load())
-	}, 30*time.Second, 10*time.Millisecond)
+		assert.GreaterOrEqual(c, r.triggered.Load(), int64(1))
```
Comment (artursouza) [inline tests/integration/suite/actors/reminders/scheduler/remove.go:149]: Why equal is not enough? Is a retry kicking off behind the scenes?

File: tests/integration/suite/daprd/jobs/remove.go [hunk 2]
```diff
@@ -114,13 +114,13 @@ func (r *remove) Run(t *testing.T, ctx context.Context) {
...
 	assert.EventuallyWithT(t, func(c *assert.CollectT) {
-		keys, rerr := etcdClient.ListAllKeys(ctx, "dapr/jobs")
+		keys, rerr := etcdClient.ListAllKeys(ctx, etcdKeysPrefix)
 		require.NoError(c, rerr)
 		assert.Len(c, keys, 1)
 	}, time.Second*10, 10*time.Millisecond)
 
 	assert.EventuallyWithT(t, func(c *assert.CollectT) {
-		assert.Equal(c, int64(1), r.triggered.Load())
+		assert.GreaterOrEqual(c, r.triggered.Load(), int64(1))
```
Comment (artursouza) [inline tests/integration/suite/daprd/jobs/remove.go:123]: Same.

**Review by artursouza — COMMENTED** [review]
Just a question on relaxing the assertion.

**Review by artursouza — APPROVED** [review]

**Pull request merged**